x-pack/plugins/task_manager/server/polling/observable_monitor.ts

```diff
@@ -31,6 +39,7 @@ export function createObservableMonitor<T, E>(
   return new Observable((subscriber) => {
     const subscription: Subscription = interval(heartbeatInterval)
       .pipe(
+        startWith(0),
```

Comment:
@gmmorris it seems when introducing the observable monitor that task manager wouldn't start claiming tasks until poll_interval * 2. With this change, it's now only poll_interval. I noticed this when writing some integration tests.

Reply:
Wouldn't that mean Master was broken and wouldn't be working for weeks?
What do you mean by `task manager wouldn't start claiming tasks until poll_interval * 2.`?
That wouldn't be true on master as `poll_interval` doesn't change on master....

Am I misunderstanding? 🤔

Reply:
Sorry I should have clarified. In my tests, I noticed the `updateByQuery` wasn't called until 6000ms passed instead of 3000ms when using a poll_interval of 3000. Adding this makes the call happen 3000ms after starting task manager instead of 6000ms.